packages/server/__tests__/db/userImporter._test_.js

```diff
@@ -54,17 +54,20 @@ describe('userImporter class test', () => {
             const rowsList = usersFromWorkbook(workbook);
             const ret = await userImporter.import(fixtures.users.adminUser, rowsList);
```

Comment:
I would recommend adding the following code to your context. This will eliminate the need to write test-specific code in your `UserImporter` class.
```js
// Ensure this is on top of the test file.
const sinon = require('sinon');
const email = require(../../src/lib/email);

beforeEach(() => {
    sinon.replace(email, 'sendWelcomeEmail', sinon.fake.returns('foo'));
})

afterEach(() => {
    sinon.reset();
})
```